pipelines/DarkEnergyCamera/ProcessCcd.yaml

```diff
@@ -24,8 +24,6 @@ tasks:
       file: $AP_PIPE_DIR/config/DECam/calibrate.py
       # Do not integrate file and pipeline configs until DM-31047 fixed, to
       # make it easier to check for changes on the obs side.
-      photoCal.match.referenceSelection.magLimit.fluxField: 'i_flux'
-      photoCal.match.referenceSelection.magLimit.maximum: 22.0
```

Comment:
I agree these can safely be removed because they are set via `config/DECam/calibrate.py` instead